### ISSUE TITLE
Material - readonly for input does not work correctly

### ISSUE THREAD

**Issue by klemenoslaj** [formly-js/ngx-formly#1277]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request
```

**Current behavior**
Focus animation is being triggered even when the control is in a `readonly` mode, however it does not focus an input field.

**Expected behavior**
Focus animation should not be triggered when in readonly mode, the same as if plain material component with `readonly` is being used.

**Minimal reproduction of the problem with instructions**

Execute the following command:
```
git clone git@github.com:klemenoslaj/ngx-formly-material-readonly-but-report.git && cd ngx-formly-material-readonly-but-report && yarn && ng serve
```

There are 2 input fields, first rendered as an `input` type by `formly` and second as a `custom` type.

`input` does not work correctly, while `custom` does, because I added `[readonly]="to.readonly"` to `matInput` element.
This is probably not the solution you want, since `formly` is using `formlyAttributes` to attach these attributes to host element, but that does not work for Material.

**Please tell us about your environment:**
OS X `10.13.6`
VSCode `1.28.2`
Yarn `1.12.1`
Node `10.13.0`

* **Language:** TypeScript `3.1.6`

* **Node (for AoT issues):** `10.13.0`

**Comment by klemenoslaj** [comment]
Since I think this is the primarily Material problem I opened a corresponding issue there as well: https://github.com/angular/material2/issues/14024

**Comment by klemenoslaj** [comment]
Since I think this is the primarily Material problem I opened a corresponding issue there as well: https://github.com/angular/material2/issues/14024

**Comment by aitboudad** [comment]
fixed in `4.8.2`
